io/disk/sysfs_att.py.data/sysfs_att.yaml

```diff
@@ -0,0 +1,14 @@
+disk: "sdd"
+queue_attributes: !mux
+    io_poll:
+        attribute_name: "io_poll"
```

Comment:
Looking at the test logs value of 1 for io_poll does not seem to be valid
run-queue_attributes-io_poll-b643: ERROR: [Errno 22] Invalid argument (0.31 s)

Do you want to add a try block to check if enabling polling for a particular device is supported or not?

Reply:
Will keep value 0 for now. Will be adding value 1 after getting required information from dev.